backend/src/main/kotlin/app/ehrenamtskarte/backend/stores/database/Schema.kt

```diff
@@ -30,7 +30,7 @@ class ContactEntity(id: EntityID<Int>) : IntEntity(id) {
 }
 
 object AcceptingStores : IntIdTable() {
-    val name = varchar("name", 150).nullable()
+    val name = varchar("name", 150)
```

Comment:
@Schedulaar @maxammann can I just make these changes or is there any kind of migration routine necessary?

Reply:
I don't know, how it will react. It might still work but leave the column nullable, as there might already be null values in there. We should test it I guess.

Reply:
I don't really know, we should not merge this until we know for sure. Right now we could just recreate the database. This is not that easy on production unfortunately.

Reply:
What does that mean? How do we know for sure? I mean I can also just revert these changes and keep the columns nullable, does not really hurt.